Update LSP6 Permission Guide

fhildeb opened the pull request:
Related code snippet:
- [ ] https://github.com/lukso-network/lukso-playground/pull/43

Hugoo commented on docs/learn/expert-guides/key-manager/get-controller-permissions.md:
```diff
@@ -5,185 +5,169 @@ sidebar_position: 1
 
 ## Setup
 
+First, we will install the 🛠️[`erc725.js`](../../../tools/erc725js/getting-started.md) library to easily query and decode the 🗂️[ERC725Y data storage](../../../standards/lsp-background/erc725.md#erc725y-generic-data-keyvalue-store) of the Universal Profile. You can then initialize a Universal Profile and connect to the LUKSO L16 network.
```
L16 network, really?

Hugoo reviewed (changes requested):
i think this needs some more time - i saw a "L16 network" occurence - i guess this was done too fast.

Hugoo commented on docs/learn/expert-guides/key-manager/get-controller-permissions.md:
```diff
@@ -5,185 +5,169 @@ sidebar_position: 1
 
-// step 1 - setup erc725.js
-const web3 = new Web3(RPC_ENDPOINT);
+// Set up Universal Profile on the LUKSO Testnet
```
This is not really correct - it doesn't setup anything on the testnet. Maybe more

"Instantiate erc725.js with a Universal Profile address" or smtgh makes more sense

Hugoo commented on docs/learn/expert-guides/key-manager/get-controller-permissions.md:
```diff
@@ -5,185 +5,169 @@ sidebar_position: 1
 
-Thanks to _erc725.js_, we can do that very easily. The library will do both steps for us and will returns the full list of controller addresses.
+<Details>
+    <summary>Show <code>AddressPermissions[]</code> value</summary>
```
I would keep this close to the code, it is the value of "result" (which btw could be renames to "permissions"

<img width="671" alt="image" src="https://github.com/lukso-network/docs/assets/477945/bbb208c4-0418-4ad0-b298-7a09bb2c09f1">

The output is quite small so im not sure we even need a Details box.

Btw, please use `<details>` as the issue with this is gone (revert to previous version of react mdx)

Hugoo replied:
im not sure these emojis 🗂️ are really needed here, same for 🛠️

Hugoo commented on docs/learn/expert-guides/key-manager/get-controller-permissions.md:
````diff
@@ -5,185 +5,169 @@ sidebar_position: 1
+<Details>
+    <summary>Show <code>decodedPermission</code> value</summary>
+
+```js
+Decoded permission for 0x5F606b5b237623463a90F63230F9b929321dbCBa = {
+  "CHANGEOWNER": true,
+  "ADDCONTROLLER": true,
+  "EDITPERMISSIONS": true,
+  "ADDEXTENSIONS": true,
+  "CHANGEEXTENSIONS": true,
+  "ADDUNIVERSALRECEIVERDELEGATE": true,
+  "CHANGEUNIVERSALRECEIVERDELEGATE": true,
+  "REENTRANCY": false,
+  "SUPER_TRANSFERVALUE": true,
+  "TRANSFERVALUE": true,
+  "SUPER_CALL": true,
+  "CALL": true,
+  "SUPER_STATICCALL": true,
+  "STATICCALL": true,
+  "SUPER_DELEGATECALL": false,
+  "DELEGATECALL": false,
+  "DEPLOY": true,
+  "SUPER_SETDATA": true,
+  "SETDATA": true,
+  "ENCRYPT": true,
+  "DECRYPT": true,
+  "SIGN": true,
+}
+```
````
is this js code? 🤔 

<img width="107" alt="image" src="https://github.com/lukso-network/docs/assets/477945/f538abbd-8521-41a4-9ede-dd30a7b20a3b">

<img width="697" alt="image" src="https://github.com/lukso-network/docs/assets/477945/2836d677-bea9-4a5b-8ac2-1873c4ae098d">

fhildeb commented:
Thanks for the feedback. I should not have missed the L16 appearance. Will remind myself to do better self-reviews.

Applied all suggestions, changed naming, and included small outputs into the code 👍🏻
Also updated the related playground code ✅
